### PR TITLE
feat: add vertical center support

### PR DESCRIPTION
**Pull request by FlurryNight**
## Summary
This PR adds vertical center support in usePosition hook (#1852)

The tooltip can be flipped to `'right' | 'left'` and the tail points correctly.

See the following examples
#### Left Tooltip
 - `<Tooltip verticalAlign='center' horizontalAlign='end' flip='left' anchorRef={tipRef} content={t(tooltip)} />`
 
#### Right Tooltip
 - `<Tooltip verticalAlign='center' horizontalAlign='start' flip='right' anchorRef={tipRef} content={t(tooltip)} />`

## Testing
Locally Tested
logto-tail-left
logto-tail-right

### CODE REVIEW

**Comment by charIeszhao** [comment]
Awesome, @FlurryNight 
We are currently celebrating the national holidays this entire week, so a delay will be expected to have it reviewed.

Sorry about that and thank you again for your contribution and huge support!

Best,
Charles

**Comment by wangsijie** [comment]
hi @FlurryNight 
Overall looks good to me! Would you like to fix the lint issues? Most of them can be auto fixed by `--fix`.

**Comment by wangsijie** [comment]
@FlurryNight no problem, and let's wait for another approval before merge.

**Review by wangsijie — APPROVED** [review]

**Comment by charIeszhao** [comment]
I'll take a look and help fix the lint issues. No worries @FlurryNight

File: packages/console/src/components/Tooltip/index.module.scss
Comment (charIeszhao) [inline packages/console/src/components/Tooltip/index.module.scss:37]: We prefer not to use `!important`. I'll take a look if this is necessary and can be avoided

**Review by wangsijie — CHANGES_REQUESTED** [review]
The `!important` is not allowed in our rules, could you find another way?

**Review by wangsijie — APPROVED** [review]
LGTM

**Pull request merged**
